tests/ord.test.js

```diff
@@ -0,0 +1,137 @@
+const cds = require("@sap/cds");
```

Comment:
Good to see, we have unittest already. I would suggest that puts the unittest file into a folder named "unittest". Because, we need to have integration test as well.

Reply:
I think jest has some conventions like having the folder name of `__tests__` (or similar). Maybe we can distinguish the different categories of tests by adding tags or prefixing the describe block with the kind of test it is. 
Then it should be possible to filter and only run a specific kind of test.

Reply:
I renamed with the folder to `__tests__` since this is the convention.
I would agree with @Fannon to use tags or prefixes in describe blocks instead.

Reply:
> I think jest has some conventions like having the folder name of `__tests__` (or similar). Maybe we can distinguish the different categories of tests by adding tags or prefixing the describe block with the kind of test it is. 
> 
> Then it should be possible to filter and only run a specific kind of test. 

In this pr, without unit test folder would fine. But I would prefer fully distinguishing unit test, integration test and e2e test. We can use tag to call some reusable tests, however we can put them into common utils. In principle, those three type of tests should have clear different logic and targets. But we can achieve it in following pr.

Reply:
Agree. There's also a 4th kind of test in my experience: The API tests, which use a request library to call the REST APIs from the outside. Or is this what you already had in mind with e2e? 
With e2e, I'm always thinking of UI testing, like cypress :)